plugin/src/main/java/com/denizenscript/denizen/events/world/PortalCreateScriptEvent.java

```diff
@@ -50,7 +50,7 @@ public boolean couldMatch(ScriptPath path) {
 
     @Override
     public boolean matches(ScriptPath path) {
-        if (!runGenericCheck(path.eventArgLowerAt(2).equals("because") ? path.eventArgLowerAt(3) : path.eventArgLowerAt(5), reason.asString())) {
+      if (path.eventArgLowerAt(2).equals("because") && !runGenericCheck(path.eventArgLowerAt(3), reason.asString())) {

```

Comment:
Looks like you accidentally broke the spacing in front of the line

Reply:
Oops, sorry. I had Google Code Style running, which does two space indentation. I push fix in a moment.